rasa/core/training/interactive.py

```diff
@@ -1130,9 +1131,22 @@ async def _validate_action(
 
     Returns `True` if the prediction is correct, `False` otherwise."""
 
-    question = questionary.confirm(f"The bot wants to run '{action_name}', correct?")
-
-    is_correct = await _ask_questions(question, conversation_id, endpoint)
+    if action_name == ACTION_UNLIKELY_INTENT_NAME:
+        print(
+            f"The bot wants to run '{action_name}' "
+            f"(https://rasa.com/docs/rasa/default-actions#action_unlikely_intent) "
+            f"to indicate that this user message was unexpected "
+            f"given the context of the conversation. "
+            f"Check out IntentTEDPolicy to learn more. "
+            f"Press any key to continue…"
+        )
+        input()
```

Comment:
how about using `questionary` here?

Reply:
From the description: "Users can then choose to confirm / deny the action prediction as usual but it doesn't really matter as running this action doesn't have any implications."
@wochinge  So my question is: if the answer doesn't matter, why do we need to ask a question?

Reply:
I see your point. I'd asking the question makes sense as
* aligns with behavior for other predicted actions
* makes uses explicitly think about what happened. If it's just a print statement they might just skim over it without actually considering the implications.